python/lib/scanstsv.py

```diff
@@ -68,7 +68,7 @@ def get_acquisition_time(self):
         """
 
         if 'acq_time' in self.tsv_headers:
-            eeg_acq_time = self.tsv_entries['acq_time']
+            eeg_acq_time = self.tsv_headers['acq_time']
```

Comment:
@cmadjar 's review will be interesting to have on this.

Reply:
```suggestion
            eeg_acq_time = self.tsv_entries['acq_time']
```

`self.tsv_entries` contains the values present in the TSV file while `self.tsv_header` contains only the header. So here, instead of returning the actual date of acquisition, it will return the header label, a.k.a. `acq_time`

Note: both `self.tsv_entries` and `self.tsv_header` are defined in the `__init__` function of the class and available across all functions of the class.

Reply:
right actually the `if` statement should be changed to that instead:

```suggestion
        if 'acq_time' in self.acquisition_data:
            eeg_acq_time = self.acquisition_data['acq_time']
```

Just tested it locally and it would work for tsv files with several entries as well as with TSV files with one entry only. 

Congratulations, you found a bug! ;)

Reply:
@h-karim forgot to tag you

Reply:
Done!